sample-texts.json

```diff
@@ -192,6 +197,26 @@
    "title": "<span style=\"font-weight: 200;\">隻铪锖锠鐖鐗</span>",
    "paragraph_font": "fonts/NotoSansSC[wght]_2.ift.woff2",
    "paragraph": [
-     "<span style=\"font-weight: 100;\">賂賠趄讚诬訫</span>"
+     "<span style=\"font-weight: 100;\">",
+      "魃魈魁鬾魑魅魍魉<br>",
+      "又双叒叕<br>",
+      "火炎焱燚<br>",
+      "水沝淼㵘<br>",
+      "土圭垚㙓<br>",
+      "犇骉羴麤<br>",
+      "猋驫鱻蟲<br>",
+      "茕茕孑立 沆瀣一气<br>",
+      "踽踽独行 醍醐灌顶<br>",
+      "绵绵瓜瓞 奉为圭臬<br>",
+      "龙行龘龘 犄角旮旯<br>",
+      "娉婷袅娜 涕泗滂沱<br>",
+      "呶呶不休 不稂不莠<br>",
+      "咄嗟 蹀躞 耄耋 饕餮<br>",
+      "囹圄 蘡薁 觊觎 龃龉<br>",
+      "狖轭鼯轩 怙恶不悛<br>",
+      "其靁虺虺 腌臢孑孓<br>",
+      "陟罚臧否 针砭时弊<br>",
+      "鳞次栉比 一张一翕<br>",
+       "</span>"
```

Comment:
There are some Chinese characters that can't be displayed and I don't know why.

![image](https://github.com/garretrieger/ift-demo/assets/2784308/cae5cecc-4f31-468a-a4e9-d38439692a8f)

Reply:
I'm seeing this too, I'll investigate and fix before I merge this.

Reply:
Ok it's been fixed on main.

Reply:
What's the cause of this?

Reply:
The IFT font encoding was missing some of the codepoints, I updated to add the missing stuff.